pkg/controller/kibana/driver_test.go

```diff
@@ -198,7 +197,7 @@ func TestDriverDeploymentParams(t *testing.T) {
 			kb := tt.args.kb()
```

Comment:
there's a `kbtype.SchemeBuilder.AddToScheme(s); ` above that we dont need anymore

Reply:
done!